packages/use-solana/src/providers.tsx

```diff
@@ -101,6 +104,14 @@ export const DEFAULT_WALLET_PROVIDERS: WalletProviderMap<
 
     isInstalled: () => window.solong !== undefined,
   },
+  [DefaultWalletType.Exodus]: {
+    name: "Exodus",
+    url: "https://www.exodus.com/browser-extension",
+    icon: EXODUS,
+    makeAdapter: () => new SolanaWalletAdapter(new ExodusWalletAdapter()),
+    isInstalled: () => window.exodus?.solana !== undefined,
+    isMobile: true,
```

Comment:
I noticed that many similar extensions have this flag (`isMobile`) set. What does it mean? Trying to figure out whether we need it for Exodus or not.

Reply:
does your adapter work on a mobile browser?